Week1/exercise/w1/index.js

```diff
@@ -17,6 +28,22 @@ console.log('header: ', header);
 // ====================================== //
 
 // BONUS:
+const addTodoButtonRef = document.querySelector("#btn-addTodo");
+console.log("addTodoButtonRef");
+const addTodo = () => {
+  const getTodoListUlRef = document.getElementById("todoList");
+  const getTodoInputRef = document.getElementById("todoInput");
+  console.log("getTodoInputRef.value");
```

Comment:
here you are logging a string while you should log the value of the input element